test/domains/vass/domain-delete.chpl

```diff
@@ -0,0 +1,4 @@
+// formerly BenA's types/records/compilerErrors/deleteRecordMessage
```

Comment:
Arrays and domains are not "records", as far as the user is concerned. This is so according to the current language definition and I do not see that changing any time soon.

I moved the tests in b78ba51